### PR TITLE
Struct AND Method support

### PR DESCRIPTION
**Pull request by go-wyvern**

### CODE REVIEW

**Comment by xushiwei** [comment]
https://github.com/goplus/gop/pull/536

**Comment by xushiwei** [comment]
merge into branch `struct_support`

File: exec/bytecode/module.go [hunk 1]
```diff
@@ -54,6 +59,11 @@ func (p *Package) NewComprehension(out reflect.Type) exec.Comprehension {
 	return NewComprehension(out)
 }
 
+// NewMethod create a Go+ function.
```
[the remark this answers is not in the document]
Comment (go-wyvern) [reply exec/bytecode/module.go:62]: I see , although I used the new function NewMethod, I still return FuncInfo

[the remark this answers is not in the document]
Comment (go-wyvern) [reply exec/bytecode/module.go:62]: I mentioned a new pr, which only contains struct:
https://github.com/goplus/gop/pull/538

In addition, the custom type and method will be put into the next pr

I agree that there is no method in bytecode, only function, but what I did was not as thorough as you said. Although I was in bytecode, NewMethod appeared in it, but in the end I reused the FuncInfo object.

I can change it more thoroughly next time pr. . Thanks comments

**Comment by xushiwei** [comment]
https://github.com/goplus/gop/pull/538

**Comment by go-wyvern** [comment]
> #538 merged

Okey, Custom type and method I will submit pr tomorrow, now this pr closed

**Pull request closed without merging**